enterprise_learner_portal/api/v1/views.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+
+from django.contrib.auth import get_user_model
+from django.core.exceptions import PermissionDenied
+from django.shortcuts import get_object_or_404
+from django.utils.translation import ugettext as _
+
+from rest_framework import permissions
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from edx_rest_framework_extensions.auth.bearer.authentication import BearerAuthentication
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+
+from enterprise.models import EnterpriseCourseEnrollment, EnterpriseCustomerUser
+from enterprise.utils import NotConnectedToOpenEdX
+from enterprise_learner_portal.api.v1.serializers import EnterpriseCourseEnrollmentSerializer
+
+try:
+    from openedx.core.djangoapps.content.course_overviews.api import get_course_overviews
+except ImportError:
+    get_course_overviews = None
+
+User = get_user_model()
+
+
+class EnterpriseCourseEnrollmentView(APIView):
+    permission_classes = (permissions.IsAuthenticated,)
+    authentication_classes = (JwtAuthentication, BearerAuthentication, SessionAuthentication,)
+
+    def get(self, request, user_email):
+        if not request.user.is_staff:
```

Comment:
This will prevent non-staff users from using this endpoint. Keep it simple for now, just return enrollments for the request user. We can make this endpoint more flexible in the future when/if we need to.

Reply:
Can you help me to understand the value of changing what I've already written here? This seems to be more of a matter of preference

Reply:
Actually @brittneyexline , do you have any sense of future product requirements that could guide this decision?

If we don't think we'll ever benefit from the way I've written this endpoint (specifically: being able to provide the email for any user), then I can change it

Reply:
The way you have written it here will prevent this endpoint from being useable by the learner portal. Most enterprise learners will not be staff so when the frontend app makes a request to this endpoint they will get a 403 response.